testgrid/cmd/transfigure/main.go

```diff
@@ -17,15 +17,26 @@ limitations under the License.
 package main
```

Comment:
@chases2 , please review the changes under transfigure directory

Reply:
Transfigure.sh is [deprecated](https://github.com/kubernetes/test-infra/blob/master/testgrid/cmd/transfigure/README.md), but it is still in use by [some users](https://github.com/kubernetes/test-infra/pulls?q=is%3Apr+is%3Aopen+transfigure.sh). These build changes look good to me, thanks!